src/meta/processors/admin/HBProcessor.cpp

```diff
@@ -39,16 +39,39 @@ void HBProcessor::process(const cpp2::HBReq& req) {
     }
 
     LOG(INFO) << "Receive heartbeat from " << host;
-    HostInfo info(time::WallClock::fastNowInMilliSec());
-    auto ret = ActiveHostsMan::updateHostInfo(kvstore_, host, info);
+    std::vector<kvstore::KV> data;
+    auto ret = kvstore::ResultCode::SUCCEEDED;
+    if (req.host.ip != -1 && req.host.port != -1) {
+        TimeInfo hbInfo(time::WallClock::fastNowInMilliSec());
+        ret = ActiveHostsMan::updateHostInfo(this->kvstore_, host, hbInfo);
+
+        if (req.__isset.leader_parts) {
+            nebula::cpp2::HostAddr storageNode;
+            storageNode.set_ip(req.host.ip);
+            storageNode.set_port(req.host.port);
+            const auto& leaderIds = std::move(req).get_leader_parts();
```

Comment:
Seems not need std::move, What do you think?

Reply:
OK!